src/Utils.js

```diff
@@ -128,3 +136,46 @@ export function AND(fns) {
     return true;
   };
 }
+
+export const attributePropMap = {
```

Comment:
how about take it from react itself? You can import HTMLDOMPropertyConfig and SVGDOMPropertyConfig from renderers/dom/shared. I think it's better, because when react team add new property it starts working here without any changes.